### PR TITLE
allow for multiple config retreval

### PR DESCRIPTION
**Pull request by antedebaas**
I need to get all mailbox configs, but the number will vary depending on the installation.
this adds a new function that allows to get all configured mailboxes

### CODE REVIEW

File: Service/Imap.php [hunk 1]
```diff
@@ -92,6 +105,41 @@ protected function getMailbox(string $name): Mailbox
         );
     }
 
+
+    /**
+     * Get new mailboxes instance.
+     *
```
Comment (Chris53897) [inline Service/Imap.php:115]: I think this could be added to both functions
@return Mailbox[]

Comment (antedebaas) [reply Service/Imap.php:115]: added

**Comment by antedebaas** [comment]
restarting from a branch

**Comment by antedebaas** [comment]
auto closed by github

**Pull request closed without merging**